.travis.yml

```diff
@@ -8,4 +8,4 @@ script:
   - carthage bootstrap --platform iOS
   - xcodebuild clean build -sdk iphonesimulator -project Issue\ Manager.xcodeproj/ -scheme Issue\ Manager CODE_SIGNING_REQUIRED=NO
 after_failure:
-  - find /var/ -name carthage-xcodebuild.*.log -exec cat {} "+"
+  - find /var -name carthage-xcodebuild.*.log -exec grep SimplePDFKit {} "+" 2>/dev/null
```

Comment:
Instead of after_failure; isn't there a way to get carthage to print the error directory to the console? I found there is a ` --verbose` option. Otherwise, try to reproduce the travis environment locally

Reply:
Damn, all this effort and I never tried the obvious solution of `--verbose`—that's exactly what I need thanks to https://github.com/Carthage/Carthage/issues/273. Now if only I could find the thread that recommended this workaround, I could tell them about it :p
Thanks!

Reply:
For the record, I tried creating a docker instance from the travis template, but was able to run the build command perfectly fine on there, i.e. I was unable to reproduce the issues…